src/maker/mod.rs

```diff
@@ -20,12 +20,12 @@ use tokio::{
     time::sleep,
 };
 
-pub use maker::{Maker, MakerBehavior};
+pub use server::{Maker, MakerBehavior};
```

Comment:
These needs to be reversed. The `main` function should be in `server.rs` and `mod.rs` should contain what `server.rs` contains now, the `Maker` structs and apis.